content/doc/book/pipeline/getting-started.adoc

```diff
@@ -22,106 +23,198 @@ This section introduces some of the key concepts to Jenkins Pipeline and
+*try sample Pipeline* option at the top right of the *Script* text area. Be
+aware that there are no canned Declarative Pipeline examples available from this
+field.
+. Click *Save* to open the Pipeline project/item view page.
+. On this page, click *Build Now* on the left to run the Pipeline.
++
+[.boxshadow]
+image:pipeline/classic-ui-left-column-on-item.png[alt="Classic UI left column on an item",width=35%]
+. Under *Build History* on the left, click *#1* to access the details for this
+  particular Pipeline run.
+. Click *Console Output* to see the full output from the Pipeline run. The
+  following output shows a successful run of your Pipeline.
++
+[.boxshadow]
+image:pipeline/hello-world-console-output.png[alt="*Console Output* for the Pipeline",width=70%]
++
+*Notes:*
+
+* You can also access the console output directly from the Dashboard by clicking
+  the colored globe to the left of the build number (e.g. *#1*).
+* Defining a Pipeline through the classic UI is convenient for testing out
+  Pipeline code snippets, or for handling simple Pipelines or Pipelines that do
+  not require source code to be checked out/cloned from a repository. As
+  mentioned above, unlike `Jenkinsfile`&zwj;s you define through Blue Ocean
+  (<<#through-blue-ocean,above>>) or in source control
+  (<<#defining-a-pipeline-in-scm,below>>), `Jenkinsfile`&zwj;s entered into the
+  *Script* text area area of Pipeline projects are stored by Jenkins itself,
+  within the Jenkins home directory. Therefore, for greater control and
+  flexibility over your Pipeline, particularly for projects in source control
+  that are likely to gain complexity, it is recommended that you use
+  <<#through-blue-ocean,Blue Ocean>> or
+  <<#defining-a-pipeline-in-scm,source control>> to define your `Jenkinsfile`.
 
 
 // Despite :sectanchors:, explicitly defining an anchor because it will be
 // referenced from other documents
 [[defining-a-pipeline-in-scm]]
-=== Defining a Pipeline in SCM
-
-Complex Pipelines are hard to write and maintain within the text area of the
-Pipeline configuration page. To make this easier, Pipeline can also be written
-in a text editor and checked into source control as a `Jenkinsfile` which
-Jenkins can load via the *Pipeline Script from SCM* option.
-
-To do this, select *Pipeline script from SCM* when defining the Pipeline.
-
-With the *Pipeline script from SCM* option selected, you do not enter any Groovy
-code in the Jenkins UI; you just indicate by specifying a path where in source
-code you want to retrieve the pipeline from. When you update the designated
-repository, a new build is triggered, as long as the Pipeline is configured with an
-SCM polling trigger.
+=== In SCM
+
+Complex Pipelines are difficult to write and maintain within the
+<<#through-the-classic-ui,classic UI's>> *Script* text area of the Pipeline
+configuration page.
+
+To make this easier, your Pipeline's `Jenkinsfile` can be written up in a text
```

Comment:
Consider

> stored

instead of

> written up

for easier reading by those who struggle with phrasal verbs

Reply:
I just left this as "written" (i.e. removing the "up") as it seem a bit unusual if text editors -
 especially very basic ones - were considered to store Jenkinsfiles. I gather this is less of an issue with more sophisticated IDEs.

I agree, minimising phrasal verb usage is better, especially for people in which English is not their first language.

I remember a German friend of mine once told me that in German, phrasal verbs tend to wrap around objective clauses, so while in English, you might have:

> They _broke up_ the argument that was getting out of hand.

In German, this might be:

> They _broke_ the argument that was getting out of hand _up_.

Ah, the confusion this could ensue.